cpp/ql/src/semmle/code/cpp/exprs/BuiltInOperations.qll

```diff
@@ -210,10 +210,10 @@ class BuiltInOperationIsUnion extends BuiltInOperation, @isunionexpr {
 }
 
 /**
- * A C++ `__builtin_types` expression (used by some implementations of the type_traits header).
+ * A C++ `__builtin_types_compatible_p` expression (used by some implementations of the type_traits header).
  */
-class BuiltInOperationBuiltInTypes extends BuiltInOperation, @typescompexpr {
-  override string toString() { result = "__builtin_types" }
+class BuiltInOperationBuiltInTypesCompatibleP extends BuiltInOperation, @typescompexpr {
```

Comment:
I'm probably being overly paranoid here, but since the name `BuiltInOperationBuiltInTypes` was part of the supported API, we shouldn't make that name go away without deprecating it for a few releases first. I recommend adding an alias from the old name to the new name, something like: 
```
  /**
   * DEPRECATED: Use `BuiltInOperationBuiltInTypesCompatibleP instead.instead.
   */
  deprecated
  class BuiltInOperationBuiltInTypes = BuiltInOperationBuiltInTypesCompatibleP;this.getChild(0) }
```

Reply:
Good idea, I'll add that.